lib/logger.js

```diff
@@ -81,9 +81,20 @@ function createLogger (options) {
     logger.child = () => logger
     return { logger, hasLogger: false }
   } else {
-    options.logger = typeof options.logger === 'object' ? options.logger : {}
-    options.logger.level = options.logger.level || 'info'
-    options.logger.serializers = Object.assign({}, serializers, options.logger.serializers)
+    const ops = {}
```

Comment:
```suggestion
    const localLoggerOptions = {}
```

^ this is clearer than having to look back to see what `ops` is.

Reply:
thanks, I got it